interface/forms/procedure_order/new.php

```diff
@@ -392,8 +411,9 @@ function validate(f) {
 
 <body class="body_top">
 
+<div class="span9 centered">
```

Comment:
Where does `span9` originate? It's not part of the bootstrap core grid system. Suggest instead using `container` and `col-md-6` (Guessing on the 6, would easily be different and then adding `col-md-offset-3` to center it.